packages/gatsby-theme-newrelic/gatsby/on-pre-render-html.js

```diff
@@ -18,6 +18,11 @@ const onPreRenderHTML = ({ getHeadComponents, replaceHeadComponents }) => {
       rel="stylesheet"
       href="https://fonts.googleapis.com/css2?family=Ovo&display=swap"
     />,
+    <script
+      type="text/javascript"
+      src="//cdn.bizible.com/scripts/bizible.js"
+      async=""
```

Comment:
Is `async` enough without the `=""`? Or does this need to include a value?

Reply:
I copied it from their [documentation](https://docs.marketo.com/display/BIZ/Adding+Bizible+Script) and I wasn't sure if simply adding `async` would turn it into `async=true` rather than an empty string. 🤷‍♂️